changelogs/fragments/394-terraform-add-config_file.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+- terraform - Adds option ``variables_files`` for multiple var-files (https://github.com/ansible-collections/community.general/issues/224).
+- terraform - Adds option ``backend_config_files``. This can accept a list of paths to multiple configuration files.
```

Comment:
```suggestion
- terraform - Adds option ``backend_config_files``. This can accept a list of paths to multiple configuration files (https://github.com/ansible-collections/community.general/pull/394).
```